bin/next-dev

```diff
@@ -61,6 +61,10 @@ srv.start(argv.port)
   }
 })
 .catch((err) => {
-  console.error(err)
+  if (err.code === 'EADDRINUSE') {
+    console.error(`Port ${argv.port} is already in use.\nUse \`npm run dev -- -p <some other port>\`.`)
+  } else {
+    console.error(err)
+  }
   process.exit(1)
```

Comment:
http://stackoverflow.com/questions/21409878/console-warning-not-displayed-when-performing-exit-in-node-js

Since console is async, there can be a race condition where your log gets eaten up by the process exiting,
to fix that you should add:
```js
process.nextTick(function() { process.exit(1) })
```
or 
```js
setTimeout(function() { process.exit(1) }, 0)
```
?